core/src/main/java/org/lflang/generator/c/CGenerator.java

```diff
@@ -1957,7 +1957,10 @@ protected void setUpGeneralParameters() {
         "LOG_LEVEL", String.valueOf(targetConfig.logLevel.ordinal()));
     targetConfig.compileAdditionalSources.addAll(CCoreFilesUtils.getCTargetSrc());
     // Create the main reactor instance if there is a main reactor.
-    createMainReactorInstance();
+    // FIXME: is `hasDeadlines` here always false? That does not look right.
```

Comment:
This `FIXME` should not be left unaddressed. Please investigate.

Reply:
I have determined that `hasDeadlines` is an unused global variable from legacy code. It is now removed.